### PR TITLE
Update 2d_body_keypoint.md

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

**Comment by jin-s13** [comment]
To avoid misunderstanding. Fix #454

File: docs/tasks/2d_body_keypoint.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ mmpose
 ```
 
 For [CrowdPose](https://github.com/Jeff-sjtu/CrowdPose) data, please download from [CrowdPose](https://github.com/Jeff-sjtu/CrowdPose).
-Please download the annotation files from [crowdpose_annotations](https://download.openmmlab.com/mmpose/datasets/crowdpose_annotations.tar).
+Please download the annotation files and human detection results from [crowdpose_annotations](https://download.openmmlab.com/mmpose/datasets/crowdpose_annotations.tar).
```
Comment (innerlee) [inline docs/tasks/2d_body_keypoint.md:212]: `results` means bbox or something else?

Comment (jin-s13) [reply docs/tasks/2d_body_keypoint.md:212]: only bbox

Comment (innerlee) [reply docs/tasks/2d_body_keypoint.md:212]: ok

**Review by innerlee — APPROVED** [review]

**Pull request merged**